velox/functions/prestosql/window/tests/NthValueTest.cpp

```diff
@@ -23,66 +23,69 @@ namespace facebook::velox::window::test {
 
 namespace {
 
+// This test class is for different variations of nth_value function
+// parameterized by the over clause. Each function invocation tests the function
+// over all possible frame clauses.
 class NthValueTest : public WindowTestBase {
  protected:
-  NthValueTest() : frameClause_("range unbounded preceding") {}
+  NthValueTest() : overClause_("") {}
```

Comment:
Is this needed?

Reply:
There are some non-parameterized NthValueTests in the last section of this file that require this default constructor. Without this constructor the compilation fails.